en/advanced_config/ethernet_setup.md

```diff
@@ -203,10 +203,10 @@ However this is not recommended because the default configuration is optimised f
 
 
 ## ROS2 Setup Example
-
-Prerequisites:
-
-- You have a supported autopilot hardware with RTPS feature enabled firmware on it by using [this guide](../middleware/micrortps.md#client-px4-px4-autopilot).
```

Comment:
Note, touched this because the document refers to RTPS. Need to verify status for XRCE microDDS and microROS.